### DISCUSSION TITLE
Only Pin Tabs

### DISCUSSION THREAD

**Discussion by Neikon** [mbnuqw/sidebery#1694] [Q&A] [answered]
Can I have only pinned tabs without any other tab?
When I close the last tab, it will automatically open a new tab 
![imagen](https://github.com/mbnuqw/sidebery/assets/1765319/be486bcc-ce81-4e9d-83dd-0ee1a0f85e5a)

**Comment by Naezr** [comment, marked as answer]
I decided to browse the Sidebery community and came across a familiar nickname. Hi!

Right click on blank space > configure panel
![image](https://github.com/user-attachments/assets/029f8d2b-cf95-488b-825e-97d5ac61d536)

**Reply by Neikon** [reply]
The world is a small world, thank you very much, I was not able to find it.
